lib/query.js

```diff
@@ -66,6 +66,46 @@ function queryByRelationship(options, cb) {
   });
 }
 
+function queryByRelationships(options, cb) {
```

Comment:
queryByRelationship (den tidigare funktionen) borde kunna vara ett anrop till queryByRelationships.

queryByRelationships borde dokumenteras i README, är det en OR eller AND när flera relationer är specade? det borde också klargöras med ett till test